src/services/address_services.py

```diff
@@ -0,0 +1,105 @@
+from src.db.address_db import fake_address_db
+from src.db.cutsomer_db import fake_customer_db
+from src.models.address import Address, AddressIn
+
+def create_address(user: AddressIn) -> dict:
+    """
+    Create new customer address,
+    return customers addresses information
+
+    Args:
+        user (UserAddress): customer address
+
+    Returns:
+        dict: customers addresses information
+    """
+    #new address id
+    address_id = fake_address_db[-1]['id'] + 1
+    #create and add address to address table
+    new_address = user.dict()
+    new_address['id'] = address_id
+    fake_address_db.append(new_address)
+    return Address(**new_address)
+
+
+def get_customer_address_customer_id(get_id: id)-> dict:
+    """
+    Return customer address information by a given customer id
+
+    Args:
+        get_id (id): customer id
+
+    Returns:
+        dict: customers addresses information for a given customer id
+    """
+    for customer, index in zip(fake_customer_db, range(len(fake_customer_db))):
```

Comment:
you can use `enumerate(fake_customer_db)` and it will give the same result
```suggestion
    for customer, index in enumerate(fake_customer_db):
```